docs/src/content/docs/ko/guides/sidebar.mdx

```diff
@@ -490,17 +482,15 @@ starlight({
 <SidebarPreview
 	config={[
 		{
-			label: '가이드',
+			label: '별자리',
 			collapsed: true,
 			items: [
-				{ label: '컴포넌트', link: '/ko/guides/components/' },
-				{ label: '국제화 (i18n)', link: '/ko/guides/i18n/' },
+				{ label: '용골자리', link: '' },
+				{ label: '켄타우루스', link: '' },
 				{
-					label: '고급',
+					label: '계절별',
```

Comment:
```suggestion
					label: 'seasonal,
```

This is also a folder in an autogenerated group, so the folder name would be used.